Assets/Scripts/Models/Furniture.cs

```diff
@@ -132,9 +132,9 @@ private Furniture(Furniture other)
         UnlocalizedDescription = other.UnlocalizedDescription;
     }
 
-    public event Action<Furniture> Changed;
+    public event Action<Furniture> OnChanged;
 
-    public event Action<Furniture> Removed;
+    public event Action<Furniture> OnRemoved;
```

Comment:
Ultimately, I don't care what it is in the end, I just prefer that it is consistent. At the moment I consider this an informal standard solely through majority.

I personally think that with how our events are registered this reads better. But if a different standard is decided upon then of course we change all to whatever the standard is.